src/components/CommentList.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../api";
+import { CommentCard } from "./CommentCard";
+
+export const CommentList = ({review_id}) => {
+
+    const [listOfComments, setListOfComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        setIsLoading(true)
+        getComments(review_id).then((comments) => {
+            setListOfComments(comments)
+            setIsLoading(false)
+        });
+    }, []);
+
+    return isLoading ? (
+        <p> Loading comments... </p>
+    ) :
+    listOfComments.length === 0 ?  (<div><p> No comments yet </p>
+    </div> )
+    :
+    (
+    <section className="commentList">
+        <h3> Comments: </h3>
+        {
+            listOfComments.map(({body, votes, author, created_at, comment_id, review_id}) => {
```

Comment:
Great use of destructuring here to grab just the information you need, keeps the code really clear!